### PR TITLE
Loading component

### PR DESCRIPTION
**Pull request by ranasobeid95**
Relates #37

### CODE REVIEW

File: client/src/components/utils/Loading/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Spin, Icon } from 'antd';
+
+import 'antd/dist/antd.css';
```
Comment (ahmedisam99) [inline client/src/components/utils/Loading/index.js]: this file will be imported in the app component, so no need for it here
```suggestion

```

File: client/src/components/utils/index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import Loading from './Loading';
+
+export default Loading;
```
Comment (ahmedisam99) [inline client/src/components/utils/index.js]: use the named export for this, as others also will export their components in this file

File: client/src/components/utils/index.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (MaiUbeid) [inline client/src/components/utils/index.js]: why does eslint error happen? 
@ranasobeid95

File: client/src/components/utils/Loading/style.css [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+    width: 100%;
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 1.5rem ;
+  }
+  .span-icon {
+    color: var(--main-color--light);
+
```
Comment (MaiUbeid) [inline client/src/components/utils/Loading/style.css]: @ranasobeid95 why there are line empty?

File: client/src/components/utils/Loading/style.css [hunk 5]
```diff
@@ -0,0 +1,12 @@
+.load-Com {
+    width: 100%;
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 1.5rem ;
+  }
```
Comment (FadiAlamassi) [inline client/src/components/utils/Loading/style.css]: leave a new line after each selector 
```suggestion
  }
  
```

File: client/src/components/utils/Loading/style.css [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+  .span-icon {
```
Comment (FadiAlamassi) [inline client/src/components/utils/Loading/style.css]: Remove space
```suggestion
.span-icon {
```

File: client/src/components/utils/Loading/style.css (hunk 4 above)
Comment (FadiAlamassi) [inline client/src/components/utils/Loading/style.css]: Remove empty line

File: client/src/components/utils/index.js (hunk 3 above)
Comment (ranasobeid95) [reply client/src/components/utils/index.js]: we need many components to be exported, so I use Named Export here nested of default

**Pull request merged**